sdk/identity/Azure.Identity/api/Azure.Identity.netstandard2.0.cs

```diff
@@ -139,6 +139,7 @@ public partial class DefaultAzureCredential : Azure.Core.TokenCredential
     public partial class DefaultAzureCredentialOptions : Azure.Identity.TokenCredentialOptions
     {
         public DefaultAzureCredentialOptions() { }
+        public string ClientId { get { throw null; } set { } }
```

Comment:
I wonder if users might need to set `ClientId` on `InteractiveBrowserCredential` and `SharedTokenCacheCredential` independently. It probably doesn't matter as much since we've disabled `SharedTokenCacheCredential` by default, but you need a first party app to use the tokens VS stashes in the cache, whereas that restriction doesn't exist for `InteractiveBrowserCredential`. Thoughts?

Reply:
Since the change would be additive to the API, I think we should start here and add the individual option if we get feedback.

Reply:
discussed more offline, will make this change.